src/components/Pool/PairsList/index.tsx

```diff
@@ -141,3 +143,39 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
     </Flex>
   )
 }
+
+const DimBgContainer = styled.div`
+  width: 100%;
+
+  padding: 48px;
+  border-radius: 12px;
+  border: 1px solid #464366;
+
+  background: linear-gradient(143.3deg, rgba(46, 23, 242, 0.5) -185.11%, rgba(46, 23, 242, 0) 49.63%);
+  background: linear-gradient(113.18deg, rgba(255, 255, 255, 0.5) -0.1%, rgba(0, 0, 0, 0) 98.9%);
+  background: #3933584d;
+
+  background-blend-mode: normal, overlay, normal;
+  backdrop-filter: blur(25px);
+`
+
+const BlueButton = styled.button`
```

Comment:
I think this button is very similar to `CurrencySelect`, we must create composable components so it makes it easier to edit styles. Maybe we can have a simple BlueButton were we can apply new styles on. Unfortunately we don't have good buttons under `components/Button`, at least for this case.

Reply:
Currently refactoring the components in another PR.